kratos/utilities/entities_utilities.cpp

```diff
@@ -62,6 +62,24 @@ bool EntitityIdentifier<TEntity>::IsInitialized() const
 /***********************************************************************************/
 /***********************************************************************************/
 
+template<class TEntity>
+bool EntitityIdentifier<TEntity>::HasPrototypeEntity(typename GeometryType::Pointer pGeometry) const
+{
+    return mTypes[static_cast<std::size_t>(pGeometry->GetGeometryType())] != nullptr;
```

Comment:
this will segfault if `pGeometry` is a `nullptr`

Another reason IMO for not adding this version, the other one puts the responsibility of the nu--check on the caller (which is where I believe it should be